src/puppet-wechat.ts

```diff
@@ -1242,28 +1242,20 @@ export class PuppetWeChat extends PUPPET.Puppet {
     }
     let mediatype: WebMediaType
 
-    switch (ext) {
```

Comment:
I'd like to keep the `switch` here because it is more clear.

Sometimes verbose is our friend, to make the code more readable.